docs/api/buckets.rst

```diff
@@ -3,8 +3,7 @@
 Buckets
 #######
 
-A bucket is the parent object of collections and groups. It can be viewed as
-a namespace where all collections and groups are stored.
+A bucket is the parent object of collections and groups.
```

Comment:
Agree w/ @ametaireau - referring to it as a namespace made it immediately obvious to me as well. :smile: